### PR TITLE
[#13835] Fix Generate keys button

### LINKED ISSUE #13835: Nothing happens on press the 'Generate keys' button during account recovery with an empty Keycard (Android)
# Bug Report

## Problem 
If follow the flow `Recover with Keycard`with no seed phrase on it, nothing happens when clicking the `Generate keys` button on the screen `There's nothing to recover here`.

#### Expected behavior
Redirect to the `Get your keys` screen

#### Actual behavior

https://user-images.githubusercontent.com/67952253/185926811-afe6452a-ba1f-4775-9d48-9511267ca6af.mp4

### Reproduction

1. Install Status
2. Tap `Get started` > `Access existing keys` > `Recover with Keycard` > `Begin recovery`
3. Tap an empty Keycard (with no keys on it) to your device 
4. Tap `Generate keys`

### Additional Information

- Status version: nightly 22/08
- Operating System: Android

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by rasom**
fix #13835

status: ready

### CODE REVIEW

File: src/status_im/ui/screens/keycard/recovery/views.cljs [hunk 1]
```diff
@@ -199,44 +199,46 @@
...
+                  [react/image {:source (resources/get-image :keycard)
+                                :style  {:width  144
+                                         :height 114}}]
+                  [react/image {:source (resources/get-image :keycard-empty)
+                                :style  {:width  165
+                                         :height 110}}])]]]
+             [react/view {:margin-bottom 50}
+              [quo/button
+               {:on-press #(do
+                             (js/alert "WAIT!")
```
Comment (ibrkhalil) [inline src/status_im/ui/screens/keycard/recovery/views.cljs]: Do we need to keep this alert ?

Comment (rasom) [reply src/status_im/ui/screens/keycard/recovery/views.cljs]: We definitely don't need it. This PR is created for debugging, I haven't requested reviews yet.

**Comment by rasom** [comment]
@ibrkhalil it is ready now

**Review by Parveshdhull — APPROVED** [review]

**Comment by qoqobolo** [comment]
Thanks for the fix @rasom, PR can be merged.

**Pull request merged**